platform/integrations/slack_integration/setup.mdx

```diff
@@ -0,0 +1,163 @@
+---
+id: setup
+title: Setup Tolgee Slack Integration
+sidebar_label: Setup
+description: Learn how to install and setup Tolgee Slack app.
+---
+
+import { ScreenshotWrapper } from '../../shared/_ScreenshotWrapper';
+
+## Prerequisites
+
+Before you begin, ensure you have:
+
+- A Tolgee account with the necessary permissions to manage integrations
+- A Slack workspace where you have permission to add apps
+
+:::info Feature availability
+
+Tolgee Slack Integration is available in the Business and Enterprise plans. [Upgrade your plan](/pricing) to use this feature.
+
+If you use the self-hosted version, you need to [set up the license](/platform/self_hosting/licensing) to use this feature.
+
+:::
+
+## Setup for Cloud Users
```

Comment:
Cloud is a priority for us, so I would remove this headline. And move the steps one level higher.
The Self-hosted setup might be moved to the self-hosting section. Here we can only add link to it.